### PR TITLE
[ISSUE-456] Downgrade kind version to 0.8.1

### PR DESCRIPTION
**Pull request by safronovD**
### Issue #456

Revert kind version to 0.8.1

## PR checklist
- [x] Add link to the issue
- [x] Choose PR label
- [ ] New unit tests added
- [ ] Modified code has meaningful comments
- [ ] All TODOs are linked with the issues
- [ ] All comments are resolved

## Testing
_Link to custom CI build_

### CODE REVIEW

File: Makefile.validation [hunk 1]
```diff
@@ -143,7 +143,37 @@ kind-install: kind-build
 	kind version
 
 kind-create-cluster:
-	$(KIND) create cluster --config $(KIND_DIR)/kind.yaml --image kindest/node:v1.18.19
+	$(KIND) create cluster --config $(KIND_DIR)/kind.yaml --image kindest/node:v1.18.2
 
 kind-delete-cluster:
 	$(KIND) delete cluster
+
+sidecars-pull:
```
Comment (mishoyama) [inline Makefile.validation:151]: Please remove these changes

File: docs/e2e-testing.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
+## Requirements
```
Comment (mishoyama) [inline docs/e2e-testing.md:1]: And these one.

**Review by mishoyama — CHANGES_REQUESTED** [review]

File: Makefile.validation (hunk 1 above)
Comment (safronovD) [reply Makefile.validation]: Removed. Sorry, I'm confused in my branches

File: docs/e2e-testing.md (hunk 2 above)
Comment (safronovD) [reply docs/e2e-testing.md]: removed

**Pull request merged**